databases/config/database.js

```diff
@@ -2,11 +2,7 @@ require('dotenv').config();
 
 module.exports = {
   development: {
-    username: process.env.DB_USERNAME,
-    password: process.env.DB_PASSWORD,
-    database: process.env.DB_NAME,
-    host: process.env.DB_HOST,
-    port: process.env.DB_PORT,
+    databaseUrl: process.env.DATABASE_URL,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from individual database configuration parameters to a single `DATABASE_URL` is a good move for simplicity and maintainability. However, it's important to ensure that the `DATABASE_URL` is correctly formatted and includes all necessary information (username, password, host, port, and database name). If not, this could lead to connection issues. Consider adding error handling or validation to check the format of the `DATABASE_URL`.

```diff
+    databaseUrl: process.env.DATABASE_URL ? new URL(process.env.DATABASE_URL) : null,
```
This will throw an error if `DATABASE_URL` is not a valid URL.



<!-- This is an auto-generated comment by OSS CodeRabbit -->